### PR TITLE
Ci introduction

### PR DESCRIPTION
**Pull request by skywojciechowskim**
Summary: Unit tests enabled in rialto-ocdm CI
Type: Feature
Test Plan: UT
Jira: RIALTO-271

### CODE REVIEW

File: .github/workflows/valgrind_ut.yml [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+    branches: [ "master", "rdkcentral:master" ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+
+  # This job downloads and caches valgrind to speed up execution
+  # An updated version of valgrind is required, the default version for ubuntu-18.04 is
```
Comment (skywilliamsonl) [inline .github/workflows/valgrind_ut.yml:41]: Update this comment, probably can remove this now and just do apt install valgrind for Ubuntu 22. We should then be able to remove all this caching for valgrind. Maybe try it out after delivering this.

**Review by skywilliamsonl — APPROVED** [review]

File: .github/workflows/valgrind_ut.yml (hunk 1 above)
Comment (skywojciechowskim) [reply .github/workflows/valgrind_ut.yml]: comment removed. I will try to install valgrind using apt-get in next commit

**Review by skywilliamsonl — APPROVED** [review]

File: rialto-ocdm.supp [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+   fun:g_type_register_static
+}
+{
+   ignore_gtype_class_ref_leak
+   Memcheck:Leak
+   ...
+   fun:g_type_class_ref
+}
+{
+   ignore_gst_init_leak
```
Comment (aczs) [inline rialto-ocdm.supp:56]: Can you try calling gst_deinit at the end of test/suite? Maybe this suppresion will not be needed.

Comment (skywojciechowskim) [reply rialto-ocdm.supp:56]: I checked it now, gst_deinit doesn't help - suppression is still needed.

**Review by aczs — APPROVED** [review]

**Pull request merged**